tests/decode_images.rs

```diff
@@ -228,7 +229,55 @@ fn test_tiled_rgb_u8() {
 
 #[test]
 fn test_tiled_rect_rgb_u8() {
-    test_image_sum_u8("tiled-rect-rgb-u8.tif", ColorType::RGB(8), 62081032);
+    let file = "tiled-rect-rgb-u8.tif";
+    let expected_type = ColorType::RGB(8);
+
+    test_image_sum_u8(file, expected_type, 62081032);
```

Comment:
I modeled this modification off of the test `test_planar_rgb_u8` which checks the image sum like many of the other tests but also adds some additional logic tests.

Reply:
Could you make a dedicated test for this rather than extending an existing test?

Reply:
I split it out and also split out the planar band chunk reading bits from the `test_planar_rgb_u8` test into a separate test.